src/components/mailFilter/ActionFileinto.vue

```diff
@@ -31,18 +27,24 @@ export default {
 			required: true,
 		},
 	},
+	data() {
+		return {
+			currentSelectedMailboxId: null,
+		}
+	},
 	computed: {
 		...mapStores(useMainStore),
 		mailbox() {
 			return this.action.mailbox ?? undefined
 		},
-		mailboxes() {
-			const mailboxes = this.mainStore.getMailboxes(this.account.accountId)
-				.filter(mailbox => mailboxHasRights(mailbox, 'i'))
-
-			return mailboxes.map((mailbox) => {
-				return mailbox.displayName
-			})
+		selectedMailbox: {
+			get() {
+				return this.currentSelectedMailboxId
+			},
+			set(selectedMailboxId) {
+				logger.debug('Selected mailbox set to', selectedMailboxId)
```

Comment:
debug log?

Reply:
i added this because thats what we do on accountdefaultsettings.vue where we also use the MailboxInlinePicker.